src/Features/Core/Portable/xlf/FeaturesResources.zh-Hans.xlf

```diff
@@ -72,12 +72,12 @@ Make sure to use the "tt" specifier for languages for which it's necessary to ma
       </trans-unit>
       <trans-unit id="Add_project_reference_to_0">
         <source>Add project reference to '{0}'.</source>
-        <target state="translated">将参数引用添加到“{0}”。</target>
+        <target state="translated">添加到“{0}”的项目引用”。</target>
```

Comment:
Here are ambiguity in English: "to" here means "... reference to {0}", not "add ... to {0}".